marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/job/testFindStepResponses.sjs

```diff
@@ -137,7 +191,46 @@ function verifyPaginationInStepResponse() {
     return assertions;
 }
 
+function verifyJobStatusInStepResponse() {
```

Comment:
You don't need a whole separate test just to verify one additional column. 

What I'll typically do here is have a "basic happy path" test where I validate every detail, and I use a basic set of inputs. Then I'll have other tests with varying sets of inputs that produce different outcomes that I need to test for. 

It looks like verifyStepResponse should be that "basic happy path" test should be that test, but it's not really validating the columns - it's just verifying that the value is null. We need at least one test that verifies every column - we can't assume that any column is properly configured.